src/gis/gestures.rs

```diff
@@ -0,0 +1,40 @@
+use gio::prelude::*;
+use gtk::prelude::*;
+
+pub fn page(header: &gtk::Widget) -> gtk::Widget {
+    let description = gtk::LabelBuilder::new()
+        .wrap(true)
+        .justify(gtk::Justification::Center)
+        .label(concat!(
+            "Use four finger swip left to pen Workspaces and windows overview, four fingers swipe ",
+            "right to pen Applications, and four fingers swipe up or down to switch between ",
```

Comment:
Minor typos:
```suggestion
            "Use four finger swipe left to open Workspaces and windows overview, four fingers swipe ",
            "right to open Applications, and four fingers swipe up or down to switch between ",
```

Note: I'm not sure if the additional letters cause the lines to exceed the preferred length.

Reply:
Thanks, looks like @mmstick applied this suggestion